### PR TITLE
Image Component done

### PR DESCRIPTION
**Pull request by NileshAwachat**

### CODE REVIEW

File: src/components/Home/Button.jsx [hunk 1]
```diff
@@ -1,9 +1,38 @@
-import React from 'react'
+/* eslint-disable react/prop-types */
```
Comment (priyankababhale) [inline src/components/Home/Button.jsx:1]: delete all the button code and reset it to default component

File: src/components/Home/Image.jsx [hunk 2]
```diff
@@ -1,9 +1,17 @@
-import React from 'react'
 
-const Image = ({url, height, width}) => {
+
+import PropTypes from 'prop-types';
```
Comment (priyankababhale) [inline src/components/Home/Image.jsx:3]: delete this import statement

File: src/components/Home/Section1.jsx [hunk 3]
```diff
@@ -1,8 +1,35 @@
...
 
 const Section1 = () => {
+
+  const handleClick = ()=>{
+    
+  }
+
   return (
-    <div>Section1</div>
+    <>
```
Comment (priyankababhale) [inline src/components/Home/Section1.jsx:12]: delete code from line 12 - 30

**Review by priyankababhale — CHANGES_REQUESTED** [review]
make the suggested changes

File: src/components/Home/Button.jsx (hunk 1 above)
Comment (NileshAwachat) [reply src/components/Home/Button.jsx:1]: ok done

File: src/components/Home/Section1.jsx (hunk 3 above)
Comment (NileshAwachat) [reply src/components/Home/Section1.jsx:12]: completed should i commit and push

**Pull request closed without merging**